### DISCUSSION TITLE
when will AIO be Nextcloud Hub 9?

### DISCUSSION THREAD

**Discussion by ne0YT** [nextcloud/all-in-one#5391] [Questions] [answered]
when will AIO be Nextcloud Hub 9 (release)?

**Comment by szaimen** [comment, marked as answer]
Hi, you can subscribe to https://github.com/nextcloud/all-in-one/issues/5249 to get notified when it gets released. 

However you can install nc30 already any time by following https://github.com/nextcloud/all-in-one/discussions/5133

**Reply by ne0YT** [reply]
thank you 👌
